test/helper.rb

```diff
@@ -145,3 +129,16 @@ def full_name
     end
   end
 end
+
+require 'openssl'
+# Silence warnings on SSLSocket initialization.
+module SSLSocketFix
```

Comment:
Can you point me to where these warnings happen currently?

Reply:
This was a [Ruby 2.2 warning](https://github.com/puma/puma/runs/628616697)